[2024-06-04T12:49:03.583+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-04T12:49:03.642+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_directions.load_directions_data manual__2024-06-04T12:48:41.445735+00:00 [queued]>
[2024-06-04T12:49:03.666+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_directions.load_directions_data manual__2024-06-04T12:48:41.445735+00:00 [queued]>
[2024-06-04T12:49:03.668+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-04T12:49:03.712+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_directions_data> on 2024-06-04 12:48:41.445735+00:00
[2024-06-04T12:49:03.732+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=157) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-04T12:49:03.739+0000] {standard_task_runner.py:63} INFO - Started process 161 to run task
[2024-06-04T12:49:03.738+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'test_directions', 'load_directions_data', 'manual__2024-06-04T12:48:41.445735+00:00', '--job-id', '14', '--raw', '--subdir', 'DAGS_FOLDER/test_directions.py', '--cfg-path', '/tmp/tmpropl1j8z']
[2024-06-04T12:49:03.744+0000] {standard_task_runner.py:91} INFO - Job 14: Subtask load_directions_data
[2024-06-04T12:49:03.890+0000] {task_command.py:426} INFO - Running <TaskInstance: test_directions.load_directions_data manual__2024-06-04T12:48:41.445735+00:00 [running]> on host 6b740a15ab9e
[2024-06-04T12:49:04.148+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Matheus' AIRFLOW_CTX_DAG_ID='test_directions' AIRFLOW_CTX_TASK_ID='load_directions_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-04T12:48:41.445735+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-04T12:48:41.445735+00:00'
[2024-06-04T12:49:04.152+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-04T12:49:04.241+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-04T12:49:04.243+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-04T12:49:04.264+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=test_directions, task_id=load_directions_data, run_id=manual__2024-06-04T12:48:41.445735+00:00, execution_date=20240604T124841, start_date=20240604T124903, end_date=20240604T124904
[2024-06-04T12:49:04.322+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-04T12:49:04.359+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-04T12:49:04.363+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
